src/science/misc/cable_climate.F90

```diff
@@ -15,271 +15,475 @@
 ! Called from: cable_driver
 !
 ! History: Vanessa Haverd Jan 2015
-
+! Aug 2017: additional leaf-level met variables stored for use in optimising ratio
+! of Jmax to Vcmax
+#ifndef UM_CBL
```

Comment:
mmm - not sure I really like this in the code base at least not without the corresponding #ifdef case